tabs/pid_tuning.js

```diff
@@ -414,20 +414,27 @@ TABS.pid_tuning.initialize = function (callback) {
         curveContext.lineTo(width / 2, height);
         curveContext.stroke();
 
-        if (scaleHeight <= height / 2) {
-            curveContext.strokeStyle = '#c0c0c0';
-            curveContext.lineWidth = 4;
+    }
 
-            curveContext.beginPath();
-            curveContext.moveTo(0, height / 2 + scaleHeight);
-            curveContext.lineTo(width, height / 2 + scaleHeight);
-            curveContext.stroke();
 
+    function plotStickPosition(curveContext, rcData, rate, rcRate, rcExpo, superExpoActive, deadband, maxAngularVel, stickColor) {
```

Comment:
Did we want to move this functionality into `RateCurve.js`? `pid_tuning.js` is already way bigger than it should be, so anything that is purely rates display related should go into `RateCurve.js`.